packages/webcomponents/src/utils/utils.ts

```diff
@@ -138,6 +138,17 @@ export const MapPayoutStatusToBadge = (status: string) => {
   }
 };
 
+export const MapTerminalStatusToBadge = (status: string) => {
```

Comment:
Yeah I dont think these functions belong in this file either. I added this here because we did the same thing for `payments status` and `payouts-status`. Just following the pattern we had set, but I agree we should figure out something more consistent for this!

I would be fine with computing it in the class file - as long as we feel OK about the class / TS file computing JSX / HTML for us?